_i18n/en/_posts/2019-08-27-press-release.md

```diff
@@ -1,6 +1,6 @@
 ---
 # layout: news
-title: Press release Aug 2019
+title: Press release - CYBERG strikes new international agreements
```

Comment:
Do we need "Press release" here ❓ 
This was the docs.label before and yesterday we have to remove it 🤔

Reply:
Yep Erik asked for it in the pdf

Reply:
OK